environment.yml

```diff
@@ -4,7 +4,7 @@ channels:
   - openeye
 dependencies:
   - python=3.7
-  - jax=0.2.27
-  - jaxlib=0.1.75
```

Comment:
weird jaxlib got a huge bump it seems

Reply:
they decided to make them both use consistent version #s

Reply:
that just makes too much sense!